front/src/modules/ui/object/record-table/components/RecordTableHeader.tsx

```diff
@@ -200,7 +222,8 @@ export const RecordTableHeader = () => {
               tableColumnsByKey[column.fieldMetadataId].size +
                 (resizedFieldKey === column.fieldMetadataId
                   ? resizeFieldOffset
-                  : 0),
+                  : 0) +
+                24,
```

Comment:
where does this 24 come from, looks a bit hacky too?

Reply:
This 24 is the width of Icon and I added it because the min width of column should be increased to ensure that Icon is visible for all the columns